package.json

```diff
@@ -14,19 +14,21 @@
     "test": "mocha -R spec --timeout 10000 --require test/init.js test/*.test.js"
   },
   "dependencies": {
-    "async": "^0.9.0",
-    "debug": "^2.1.1",
-    "loopback-connector": "^2.1.0",
-    "pg": "^6.0.0",
-    "strong-globalize": "^2.6.2"
+    "async": "^0.9.2",
+    "debug": "^2.2.0",
+    "loopback-connector": "^2.4.0",
+    "pg": "^6.0.3",
+    "strong-globalize": "^2.6.6"
```

Comment:
You should be upgrading these in a separate PR. This has nothing to do with linting.